packages/firebase_ml/lib/src/model_management/firebase_model_download_conditions.dart

```diff
@@ -1,31 +1,46 @@
 /// Conditions to download remote models.
 class FirebaseModelDownloadConditions {
-  /// Boolean value that indicates if wifi is required.
-  final bool requireWifi;
+  /// Android: Boolean that indicates if wifi is required.
```

Comment:
most of this can be inferred from the actual code, I'd change it to something like this or maybe better just merge all of them into one saying they are for android only

```
/// Android only: if wifi is required to download model.
```

Reply:
https://screenshot.googleplex.com/5MackpyCuR1
from 
https://www.investigatii.md/uploads/resurse/Clean_Code.pdf

Reply:
Thank you. 
In flutter all public members have to be documented, so I will not merge all of these comments into one saying. 
I will follow your suggestion on removing redundancy in each line.